action.yml

```diff
@@ -14,7 +14,11 @@ runs:
   using: "composite"
   steps:
     - name: Fetch entire Git history (including tags)
-      run: git fetch --prune --unshallow --tags
+      run: |
+        if [ "$(git rev-parse --is-shallow-repository)" = "true" ]; then
+          git fetch --prune --unshallow
```

Comment:
Q: What's the reason for removing the ` --tags` option at this point?

Reply:
If a previously `git clone --no-tags` was made a `git fetch --tags` is needed for unshallowed repositories.